refactor(metadata): remove xml format option from import & export

Mohammer5 opened the pull request:
Closes [DHIS2-13697](https://dhis2.atlassian.net/browse/DHIS2-13697)

Removes the xml format option from the Metadata import and export forms.

tomzemp commented:
@Mohammer5 Looks good, but some questions:

do you want to update the text to remove the reference to XML: https://github.com/dhis2/import-export-app/blob/146090211b71b98401c4bd89cafc3fd664f9bf45/src/pages/MetadataExport/MetadataExport.js#L27

(the text for the import seems to be maybe wrong? https://github.com/dhis2/import-export-app/blob/d4c3aa026d7502a6fc4afe36d9cd630c1f5114f9/src/pages/MetadataImport/MetadataImport.js#L53)
The import also has help text to update though: https://github.com/dhis2/import-export-app/blob/d4c3aa026d7502a6fc4afe36d9cd630c1f5114f9/src/pages/MetadataImport/MetadataImport.js#L129

Do you want to remove the xml option from Metadata Dependency export as well?

There are some cypress tests that mention xml for import/export and I wonder if they are still relevant: https://github.com/dhis2/import-export-app/blob/7387833a2a9d36b0952542d4a6786e3d3f8605f4/cypress/integration/EndpointParamChecks/ExportMetaData.feature#L29
https://github.com/dhis2/import-export-app/blob/7387833a2a9d36b0952542d4a6786e3d3f8605f4/cypress/integration/EndpointParamChecks/ImportMetaData.feature#L28
https://github.com/dhis2/import-export-app/blob/7387833a2a9d36b0952542d4a6786e3d3f8605f4/cypress/integration/EndpointParamChecks/ExportData.feature#L52

Mohammer5 commented:
@tomzemp thanks! I've addressed the texts and removed xml from the cypress files.
It seems that the cypress tests are out of date and I can't get them to run, but I'd argue that that's a different task, so won't address here.
